packages/common/src/events/_internal/EventUiSchemaEdit.ts

```diff
@@ -105,20 +98,13 @@ export const buildUiSchema = async (
             type: "Control",
             options: {
               control: "hub-field-input-date",
-              min: minStartEndDate,
               messages: [
                 {
                   type: "ERROR",
                   keyword: "required",
                   icon: true,
                   labelKey: `${i18nScope}.fields.endDate.requiredError`,
                 },
-                {
```

Comment:
I believe we still need this `formatMinimum` error, as the `endDate` cannot be set to a value before `startDate` as enforced [here](https://github.com/Esri/hub.js/pull/1521/files#diff-d93f11408763f424fc8cbd0906480ea7eddafe78a44970eb03004286f7289a13R45). Removing this results in the following validation error message in the UI:

![2024-05-21_09-32-30](https://github.com/Esri/hub.js/assets/52967037/64da0cb2-485a-4f15-88aa-83decc72197a)

Whereas before this change it was:

![2024-05-21_09-34-34](https://github.com/Esri/hub.js/assets/52967037/2182e86c-dd03-49e3-b968-fe9e42854cf7)